fixes #16 (#17)
lamps-wg/csr-attestation: draft-ounsworth-csr-attestation.md

--- abstract

> Believable claims about a device or device components, such as hardware security modules, can help to improve the assessment of its security posture.

Author: ```suggestion
Believable claims about a device or device's components, such as hardware security modules, can help to improve the assessment of its security posture.
```

> Believable claims about a device or device's components, such as hardware security modules, can help to improve the assessment of its security posture.

Author: ```suggestion
Believable claims about a device or a device's components, such as hardware security modules, can help to improve the assessment of its security posture.
```

> Believable claims about a device or a device's components, such as hardware security modules, can help to improve the assessment of its security posture.

Reviewer: ```suggestion
Believable claims about a the protections afforded to a private key, such as the protections provided by hardware security modules, can help to improve the assessment of the security posture for the private key.
```

> Believable claims about a the protections afforded to a private key, such as the protections provided by hardware security modules, can help to improve the assessment of the security posture for the private key.

Author: ```suggestion
Believable claims about a the protections afforded to a private key, such as the protected capabilites provided by hardware security modules, can help to improve the assessment of the security posture for the private key.
```

> Believable claims about a the protections afforded to a private key, such as the protected capabilites provided by hardware security modules, can help to improve the assessment of the security posture for the private key.

Author: ```suggestion
Believable claims about the protections afforded to a private key, such as the protected capabilities provided by hardware security modules, can help to improve the assessment of the security posture for the private key.
```
